static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -115,6 +115,7 @@ $gutter-large: 55px;
 
     &--open {
```

Comment:
We should change this to be the full classname, not nested to make sure it is grepable